_posts/2023-02-14-Report-Systems-To-Users.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: post
+title: Report on Systems and Users
+description: Generate a CSV report for systems and their associated users
+tags:
+  - powershell
+  - reports
+  - automation
+---
+
+A CSV report of systems and their user associations (inactive and active). This script will generate a "Report-SystemToUserAssociation.csv" file relative to the location this script is saved and ran.
+
+### Basic Usage:
+
+* Install the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Using-the-JumpCloud-PowerShell-Module)
+* Save the contents of the script example to a file on a system.
+  * EX: ~/Report-Systems_To_User_Associations.ps1
+  * Edit the the `Global Configuration - Start` section with your APIKey, report file name, and user associations
+* In a PowerShell terminal window run:
+  * `~/Report-Systems_To_User_Associations.ps1`
```

Comment:
I think just adding a section here to state that the report file name will be saved in the current working directory would be helpful for some folks